app/services/permitted_attributes/order_cycle.rb

```diff
@@ -0,0 +1,46 @@
+module PermittedAttributes
+  class OrderCycle
+    def initialize(params)
+      @params = params
+    end
+
+    def call
+      return @params[:order_cycle] if @params[:order_cycle].empty?
+
+      @params.require(:order_cycle).permit(
+        :name, :orders_open_at, :orders_close_at, :coordinator_id,
+        :incoming_exchanges => permitted_exchange_attributes,
+        :outgoing_exchanges => permitted_exchange_attributes,
+        :schedule_ids => [], :coordinator_fee_ids => []
```

Comment:
Any reason for using the hash-rocket style here? We could do this:
```ruby
:name, :orders_open_at, :orders_close_at, :coordinator_id,
incoming_exchanges: permitted_exchange_attributes,
outgoing_exchanges: permitted_exchange_attributes,
schedule_ids: [], coordinator_fee_ids: []
```

Reply:
yeah, I thought this was causing problems somewhere, but apparently not :+1: